playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml

```diff
@@ -1,17 +1,4 @@
 ---
-# We need docker service up to remove all the images, but these services will keep
-# trying to re-start and thus re-pull the images we're trying to delete.
-- name: Stop containerized services
-  service: name={{ item }} state=stopped
-  with_items:
-  - "{{ openshift_service_type }}-master-api"
-  - "{{ openshift_service_type }}-master-controllers"
-  - "{{ openshift_service_type }}-node"
```

Comment:
-node service is system container on atomic, although run by runc via systemd.  Need to ensure logic for upgrade continues to work without stopping node service on atomic.